internal/runtime/docker/invoke.go

```diff
@@ -58,11 +58,10 @@ func (r ToolRuntime) RunWithOpts(ctx context.Context, opts rtypes.RunToolOpts, o
 
 	return tasks.Action("container.execute").
 		LogLevel(2).
-		Arg("command", opts.Command).
+		Arg("binaryConfig", opts.BinaryConfig).
```

Comment:
This means that a whole json is being printed out, that doesn't feel quite right.